tuning/sft_trainer.py

```diff
@@ -325,9 +330,7 @@ def train(
         eval_dataset=formatted_validation_dataset,
         packing=packing,
```

Comment:
Let's also remove `packing` as it is on their [deprecation list](https://github.com/huggingface/trl/blob/main/trl/trainer/sft_trainer.py#L103).